UsersApp/Networking/NetworkManager.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  NetworkManager.swift
+//  Prob
+//
+//  Created by Dar Dar on 04.10.2023.
+//
+
+import Foundation
+
+
+class NetworkManager {
+    let networError = NetworkError.self
+    
+    func getUser() async throws -> [User] {
+        let endpoint = "https://randomuser.me/api/?results=100&seed=abc"
```

Comment:
Let's separate the endpoint in URLComponents. The result and seed should be specified as separate query parameters, allowing the user to adjust the number of fetched users if needed.